HHS/OPRE-OPS
# feat: ✨ adds StatusChangeReviewCards

## What changed

- handles status change requests
- adds StatusChangeReviewCard
- adds decimal place to formatter function
- moves mocks module to tests directory

## Issue

#2307 #2441

## How to test

1. goto Agreement 1/9
2. submit a BLI for approval
3. goto Agreement Change Requests
4. see StatusChangeReviewCard

## Screenshots

https://github.com/HHS/OPRE-OPS/assets/4629398/bac100a8-e9a6-4be1-9d2c-8000e3aa0783

## Definition of Done Checklist
